# This helps you (suggest-json-response)

Please describe the origin of the rule here.

## Rule Details

This rule aims to...

Examples of **incorrect** code for this rule:

```js
```

Examples of **correct** code for this rule:

```js
```

